.env.example

```
@@ -27,10 +27,10 @@ CALCOM_PRIVATE_API_ROUTE="https://goblin.cal.com"
 # ***********************************************************************************************************
 
 # - DATABASE ************************************************************************************************
-DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
+DATABASE_URL="postgresql://postgres:@localhost:5432/calendso"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)